index.rst

```diff
@@ -1,61 +1,150 @@
+Each workflow in turn results in multiple jobs being executed on the batch systems.
+Note that the Data Release Production is not expected to be run as a single workflow submission for a variety of reasons:
+ * Quantum Graph generation currently takes a long time, particularly as the graph gets more complex; smaller graphs should be non-linearly faster to generate
+ * resource limitations such as available scratch space that may be easier for humans to manage than the workflow system
+ * global synchronization points that form natural places to break up the overall workflow
+ * user intervention points where analysis of what has been completed and potentially patching of code or configuration may be desirable before continuing
+ * allowing simpler restart after failure
 
-     Caption text.
+The exclusion list and configuration information is checked into a git repository and tagged.
+The Data Release Lead Campaign Manager, a role in the Execution team, is told to execute the campaign defined by this input data and processing specification.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+The DRLCM begins by defining the campaign in the Campaign Management system.
+A campaign identifier is automatically generated, but the DRLCM provides information such as a short name, a long description including the reason why this campaign is being run, and links to the data and processing specifications.
 
-   Feel free to delete this instructional comment.
+.. figure:: /_static/DefineCampaign.png
+   :name: fig-define-campaign
+   :alt: Sketch of a campaign definition screen.
 
-:tocdepth: 1
+   Sketch of a campaign definition screen.
+    
+After defining the campaign, the DRLCM can save it or begin execution by running the first BPS command to submit the first workflow.
+(Initially, command execution will be manual, but eventually it may be integrated into the Campaign Management system.)
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+As the campaign executes, each workflow submission can be assigned one or more Operators to ensure that it is running properly.
+Operators may take shifts, defined by a schedule.
+(Initially, Operator assignments and changes of assignment will be manually entered into the Campaign Management system, but eventually the system may allow a schedule to be defined and automatically applied.)
 
-.. sectnum::
+The progress of each workflow submission is monitored through the workflow system.
+Results of each workflow submission are placed in a Butler Repository (for data products), a metric system (like SQuaSH), or a quality assurance analysis system (for things like plots).
+The Campaign Management system provides links to each of those external systems for easy access.
```

Comment:
The capability to kill a campaign will be needed from the beginning.  Not sure if ordering/prioritizing workflows (and changing the priority of a particular workflow) will be needed early or not.  For example, will we need to order workflows to guarantee outputs for large sections of the sky instead of a "random" pattern.  Another example is wanting outputs from a particular workflow sooner rather than later for science reasons.  Other capabilities are listed in the use cases/requirements document, but many of those probably can be added later.